src/dafka_consumer.c

```diff
@@ -202,8 +210,8 @@ dafka_consumer_recv_subscriptions (dafka_consumer_t *self)
         zhashx_insert (self->sequence_index, sequence_key, &last_known_sequence);
     }
 
-    if (((id == DAFKA_PROTO_MSG || id == DAFKA_PROTO_DIRECT_MSG) && !(msg_sequence == last_known_sequence + 1)) ||
-        (id == DAFKA_PROTO_HEAD && !(msg_sequence == last_known_sequence))) {
+    if (((id == DAFKA_PROTO_MSG || id == DAFKA_PROTO_DIRECT_MSG) && (msg_sequence < last_known_sequence + 1)) ||
```

Comment:
shouldn't this be great than?

Reply:
I always get these comparison wrong :man_facepalming:

Reply:
:)

Reply:
Fixed